testing/py-cassandra-driver/APKBUILD

```diff
@@ -0,0 +1,50 @@
+# Maintainer: Austin Page <jaustinpage@gmail.com>
+# Contributor: Austin Page <jaustinpage@gmail.com>
+pkgname=py-cassandra-driver
+_pkgname=python-driver
+pkgver=3.7.0
+pkgrel=0
+pkgdesc="Python driver for Apache Cassandra"
+url="https://pypi.python.org/pypi/cassandra-driver/"
+arch="noarch"
+license="ASL 2.0"
+depends="cython libev"
+makedepends="cython-dev libev-dev py-setuptools python2-dev python3-dev"
+subpackages="py3-${pkgname/py-/}:_py3 py2-${pkgname/py-/}:_py2"
+source="${pkgname/py-/}-$pkgver.tar.gz::https://github.com/datastax/$_pkgname/archive/$pkgver.tar.gz"
+builddir="$srcdir/$_pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+	python2 setup.py build || return 1
+	python3 setup.py build || return 1	
+}
+
+package() {
+	mkdir -p "$pkgdir"
+}
+
+_py() {
+	local python=$1
+	pkgdesc="$pkgdesc - $python"
+	arch="all"
+	install_if="$pkgname=$pkgver-r$pkgrel $python" 	
+	cd "$builddir"
+	$python setup.py install --prefix=/usr --root="$subpkgdir"
+}
+
+_py2() {
+	_py python2
+	replaces="$pkgname"
+	depends=""
+}
```

Comment:
This is wrong, please follow [example on the wiki](https://wiki.alpinelinux.org/wiki/APKBUILD_examples:Python).